### PR TITLE
[CDAP-7415] fix comparison between ProgramId and Id.Program

### PR DESCRIPTION
**Pull request by ecfm**
https://issues.cask.co/browse/CDAP-7415
http://builds.cask.co/browse/CDAP-RUT221-1

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public synchronized RuntimeInfo lookup(ProgramId programId, final RunId runId) {
         continue;
       }
 
-      if (!id.toId().equals(programId)) {
+      if (!id.equals(programId)) {
```
Comment (anwar6953) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java:154]: You can ask Intellij to find more instances of this warning. I think it'd be worth it to run that check.

Comment (ecfm) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java:154]: Yes, I've run it and didn't find other occurrences.

**Comment by ecfm** [comment]
Build is green

**Comment by chtyim** [comment]
LGTM

**Pull request merged**